install.sh

```diff
@@ -167,5 +167,55 @@ runchecksonstart: true
 loglevel: error
 EOF
 
-printSuccess "The Site Wide Analysis Tool Agent has been successfully installed at $agentPath"
-[ "$installDaemon" ] && installAndConfigureDaemon || installAndConfigureCron
+echo "** Install validation "
+
+if [[ ! -f "$appRoot/app/etc/env.php" ]]; then
+  error_exit "Magento not found."
```

Comment:
Increase message verbosity

error_exit "Magento not found in the path $appRoot"